### PR TITLE
Feature: add NatsMessage ack_sync method #1906

### LINKED ISSUE #1906: Feature: add NatsMessage ack_sync method
We should add an extra `ack_sync` proxy method the same way with regular `ack` we did - https://github.com/airtai/faststream/blob/main/faststream/nats/message.py#L10

### PR DESCRIPTION
**Pull request by wpn10**
Description
Added the ack_sync method to NatsMessage to support synchronous acknowledgment functionality.

Fixes #1906

Type of Change
 New feature (a non-breaking change that adds functionality)
Checklist
 My code adheres to the style guidelines of this project (scripts/lint.sh shows no errors)
 I have conducted a self-review of my own code
 I have ensured that static analysis tests are passing by running scripts/static-analysis.sh

### CODE REVIEW

File: faststream/nats/message.py [hunk 1]
```diff
@@ -17,6 +17,11 @@ async def ack(self) -> None:
             await self.raw_message.ack()
         await super().ack()
 
+    def ack_sync(self) -> None:
+        if not self.raw_message._ackd:
+            self.raw_message.ack()
```
Comment (Lancetnik) [inline faststream/nats/message.py:22]: Can you please call `raw_message.ack_sync()` here?

Comment (wpn10) [reply faststream/nats/message.py:22]: Sure, will update.

File: faststream/nats/message.py [hunk 2]
```diff
@@ -17,6 +17,11 @@ async def ack(self) -> None:
             await self.raw_message.ack()
         await super().ack()
 
+    def ack_sync(self) -> None:
```
Comment (Lancetnik) [inline faststream/nats/message.py:20]: Sorry, didn't see this first time. `ack_sync` doesn't mean, that it is a sync method. It means, that it waits for acknowledgement confirmation frame. Please, make it `async def` with original **nats-py** method awaiting.

File: tests/brokers/nats/test_consume.py [hunk 3]
```diff
@@ -214,6 +214,34 @@ async def handler(msg: NatsMessage):
...
+        self,
+        queue: str,
+        event: asyncio.Event,
+        stream: JStream,
+    ):
+        consume_broker = self.get_broker(apply_types=True)
+
+        @consume_broker.subscriber(queue, stream=stream)
+        async def handler(msg: NatsMessage):
+            msg.ack_sync()
```
Comment (Lancetnik) [inline tests/brokers/nats/test_consume.py:227]: Should be awaited here as well

**Review by Lancetnik — CHANGES_REQUESTED** [review]
Also it will be perfect if you add any test like that https://github.com/airtai/faststream/blob/main/tests/brokers/nats/test_consume.py#L189

File: tests/brokers/nats/test_consume.py (hunk 3 above)
Comment (wpn10) [reply tests/brokers/nats/test_consume.py:227]: Got it, updating

**Comment by Lancetnik** [comment]
@wpn10 thank you a lot! Great job, you will see your changes in a next release 😄

**Pull request merged**